CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.0.7
+  - Updated riemann-client gem to 0.2.6 to benefit timeout support
```

Comment:
Minor nit: should we link the issue it fixes here? (as how we are linking other changelog entries?)

Reply:
I did but for some reason github has not seen that last commit yet. looks like github is having some issues at the moment. Will wait a bit.